applications: nrf_desktop: Add DT alias to configure QDEC in wheel (#14483)
nrfconnect/sdk-nrf: applications/nrf_desktop/doc/wheel.rst

For detecting rotation, the wheel module uses Zephyr's QDEC driver.
You can enable the module only when QDEC is configured in DTS and the Zephyr's QDEC driver is enabled with the :kconfig:option:`CONFIG_QDEC_NRFX` Kconfig option.

> If used board supports multiple QDEC instances (for example ``nrf54l15pdk_nrf54l15_cpuapp``), you also need to specify the used QDEC instance with ``nrfdesktop-wheel-qdec`` DT alias.
> If the board supports only one QDEC instance, the module relies on ``qdec`` DT label and defining the DT alias is not required.

Reviewer: ```suggestion
If your board supports multiple QDEC instances (for example ``nrf54l15pdk_nrf54l15_cpuapp``), you also need to specify the used QDEC instance with the ``nrfdesktop-wheel-qdec`` DT alias.
If your board supports only one QDEC instance, the module relies on the ``qdec`` DT label and you do not need to define the DT alias.
```
